From d07fd86d5253e6f6cab39cc792b2efbc1fcb90bc Mon Sep 17 00:00:00 2001
From: james55555 <grahamj1@aston.ac.uk>
Date: 21-Mar-2014 15:20:11

formatted

diff --git a/FinalYearProject/Includes/System/Model/Task_Model.php b/FinalYearProject/Includes/System/Model/Task_Model.php
index fe74276..2a6a08c 100644
--- a/FinalYearProject/Includes/System/Model/Task_Model.php
+++ b/FinalYearProject/Includes/System/Model/Task_Model.php
@@ -215,6 +215,7 @@
             {
             return false;
             }
+            return true;
         }
 
     /* public static
